Assignment/FSMTest.c

```diff
@@ -0,0 +1,92 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Vinitha Perumal - GET
+// ------------------------------------------------------------------
+// Test.c
+// Program on A6B branch.
+// ------------------------------------------------------------------------------------------------
+#define _CRT_SECURE_NO_WARNINGS 1
+#include <malloc.h>
+#include <stdio.h>
+#include <string.h>
+#include <windows.h>
+
+int ExecProgram (const char* exeFilePathAndName, const char* inputFilePathAndName, const char* outputFilePathAndName);
+int CompareFiles (const char* file1, const char* file2);
+
+int ExecProgram (const char* exeFilePathAndName, const char* inputFilePathAndName, const char* outputFilePathAndName) {

```

Comment:
Don't have any changes in the Exec program method, implement changes in the main or just have separate method